### ISSUE TITLE
How to make the code non-global??

### ISSUE THREAD

**Issue by wowweemip-fan** [stylish-userstyles/stylish#336]
How to make the code non-global?? It keeps saying "Style code looks unintentionally global". Smh.

**Comment by ghost** [comment]
Yeah, the instruction is very vague.  They say 'click the "Export" button, under the "Mozilla Format" heading'. What "Export button", what "Mozilla Format" heading' -- they don't explain.

**Comment by JasonHorkles** [comment]
> Include the following code anywhere in your style:
> `/* i really want this to be global */`
